lib/network/rpc/rpc.ts

```diff
@@ -1,5 +1,6 @@
 import ioClient from 'socket.io-client';
 import * as http from 'http';
+import { http as HttpAgent } from 'httpagent';
```

Comment:
is that this httpAgent is imported but not used 🤔 ?

Reply:
Indeed, I forgot to cleanup this one. I will remove it, thanks!

Done and squashed in: https://github.com/scality/Arsenal/pull/2053/commits/66a9aca08196d07ca84dc3b9f9bea37e8757b148